src/+otp/+quadratic/+presets/L96PODGROM.m

```diff
@@ -0,0 +1,35 @@
+classdef L96PODGROM < otp.quadratic.QuadraticProblem
+    
+    properties
+        Projection
+        Interpolation
+    end
+
+    methods
+        function obj = L96PODGROM(r)
```

Comment:
Maybe rename to Lorenz96PodRom to be consistent.  For example, we use Rhs instead of RHS